### PR TITLE
Define how physical channel power leaves are used.

### PR DESCRIPTION
**Pull request by ejbrever**
This change:
* Removes the "when" statement and allows the descriptions to clarify when/how the leaves should be used.
* This allows the input-power to be utilized for coherent or standard transceivers. For coherent transceivers, this allows the optical-channel to represent the signal power, while the physical-channel input-power to represent the signal+noise.

More info at:
https://github.com/openconfig/public/issues/1188

### CODE REVIEW

**Comment by ejbrever** [comment]
@dplore @ahsaanyousaf can you please review? Thanks.

**Review by ahsaanyousaf — APPROVED** [review]

**Comment by ahsaanyousaf** [comment]
Thanks for the changes, LGTM

File: release/models/platform/openconfig-platform-transceiver.yang [hunk 1]
```diff
@@ -384,24 +407,6 @@ revision "2023-08-30" {
...
-    description
-      "Extended operational state data for physical client channels
-      for applications where the full physical channel config and
-      state are used. In some cases, such as when the physical
-      channel has a leafref to an optical channel component and the
-      module-functional-type is TYPE_DIGITAL_COHERENT_OPTIC this
-      grouping will NOT be used.";
-
     uses output-optical-frequency;
     uses optical-power-state;
```
Comment (dplore) [inline release/models/platform/openconfig-platform-transceiver.yang:411]: indent should be updated

**Comment by dplore** [comment]
Setting last-call for Oct 10, 2024

**Review by dplore — APPROVED** [review]

**Pull request merged**